string2.py

```diff
@@ -4,7 +4,9 @@
 """
 # Your name, plus anyone who helped you with this assignment.
 # Give credit where credit is due.
+__author__ = "Manuel"
 __author__ = "Amanda help with front_back"
```

Comment:
This is a good improvement, however you should not have multiple __author__ strings because the second will overwrite the first.  If you want to express authorship over multiple lines, you should use the triple-quoted strings like this example.  Also use your full name, or your github username.
```
__author__ = (
    """
    menosvelasco
    Amanda Yonce (front-back help)
    other stackoverflow references or links
    """
)
```